cardinal/server/server.go

```diff
@@ -20,19 +21,21 @@ import (
 )
 
 const (
-	defaultPort     = "4040"
-	shutdownTimeout = 5 * time.Second
+	defaultPort              = "4040"
+	shutdownTimeout          = 5 * time.Second
+	defaultMessageExpiration = 10   // seconds
+	defaultHashCacheSizeKB   = 1024 // default to 1MB hash Cache
 )
 
 type config struct {
-	port                            string
-	isSignatureVerificationDisabled bool
-	isSwaggerDisabled               bool
+	port              string
+	isSwaggerDisabled bool
 }
 
 type Server struct {
 	app    *fiber.App
 	config config
+	verify handler.SignatureVerification
```

Comment:
why isn't this just a part of config?

Reply:
Because it contains a reference to the cache object, so it's more than just config data. I can add it, just seemed cleaner not to. [](edited-with-Axolo)